include/InstrumentPlayHandle.h

```diff
@@ -56,7 +56,7 @@ class EXPORT InstrumentPlayHandle : public PlayHandle
 		do
 		{
 			nphsLeft = false;
-			foreach( const NotePlayHandle * cnph, nphv )
+			for( const NotePlayHandle * cnph : nphv )
 			{
 				NotePlayHandle * nph = const_cast<NotePlayHandle *>( cnph );
```

Comment:
Is it possible to get the collection of `NotePlayHandles` that are not `const` to get rid of this `const_cast`?